AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift

```diff
@@ -17,17 +17,19 @@ class JoiningViewController: UIViewController, UITextFieldDelegate {
     @IBOutlet var callKitOptionsPicker: UIPickerView!
     @IBOutlet var audioModeOptionsPicker: UIPickerView!
     @IBOutlet var audioDeviceCapabilitiesPicker: UIPickerView!
+    @IBOutlet var reconnectTimeoutPicker: UIPickerView!
     @IBOutlet var joinButton: UIButton!
     @IBOutlet var debugSettingsButton: UIButton!
     @IBOutlet var audioRedundancySwitch: UISwitch!
 
-    var callKitOptions = ["Don't use CallKit", "CallKit as Incoming in 10s", "CallKit as Outgoing"]
-    var audioModeOptions = ["Stereo/48KHz Audio", "Mono/48KHz Audio", "Mono/16KHz Audio"]
-    var audioDeviceCapabilitiesOptions = ["Input and Output", "None", "Output Only"]
+    let callKitOptions = ["Don't use CallKit", "CallKit as Incoming in 10s", "CallKit as Outgoing"]
+    let audioModeOptions = ["Stereo/48KHz Audio", "Mono/48KHz Audio", "Mono/16KHz Audio"]
+    let audioDeviceCapabilitiesOptions = ["Input and Output", "None", "Output Only"]
+    let reconnectTimeoutOptions = [180000, 20000, 5000, 0]
```

Comment:
What happens when you choose 0? Reconnection immediately times out? Would we even want to attempt a reconnect in that case? What happens with other invalid inputs like negative numbers?

Reply:
If it's <= 0, the session will be stopped immediately.